README.md

```diff
@@ -19,10 +20,25 @@ which is also the one used by [AutoStitch](http://matthewalunbrown.com/autostitc
 Eigen, CImg and FLANN are header-only, to simplify the compilation on different platforms.
 CImg and libjpeg are only used to read and write images, so you can easily get rid of them.
 
+### Steps
+
+* Download Cmake and install the cmake
+* Download Eigen and unzip it in panorama/Eigen, please ensure that panorama/Eigen/Eigen/Dense exists.
+
 ### Compile:
 Linux / OSX (gcc or clang):
 ```
-$ cd src; make
+$ cmake .
+$ make
```

Comment:
I am not sure ,whether the original Makefile is generated by cmake, because I found you have provide a CMakeList.txt , if it is generated by cmake, can we override it ?